src/App.tsx

```diff
@@ -10,9 +11,33 @@ export const App = () => {
 
       <div className="section">
         <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
+          <Routes>
+            <Route
+              path="*"
+              element={<h1 className="title">Page not found</h1>}
+            />
+            <Route
+              path="home"
+              element={<Navigate to="/" replace />}
+            />
+
+            <Route
+              path="/"
+              element={<h1 className="title">Home Page</h1>}
+            />
+
+            <Route path="people">
```

Comment:
It's good practice to have some kind of variable that stores all route values in one place, enum, or just a const. It'll be very clear to see all existing routes in one place and also when in some other components you have to use a redirect or link you can import that variable and avoid using hardcoded values which usually cause bugs and makes refactoring harder.